node/external/blockAPI/interface.go

```diff
@@ -2,11 +2,19 @@ package blockAPI
 
 import (
 	"github.com/ElrondNetwork/elrond-go-core/data/api"
+	"github.com/ElrondNetwork/elrond-go-core/data/transaction"
 )
 
+// TransactionUnmarshalerHandler defines the transaction unmarshaler handler should do
```

Comment:
replace `unmarshaler` with `unmarshaller` in the entire PR

Reply:
renamed.